pytest/tests/stress/stress.py

```diff
@@ -79,9 +77,8 @@ def stress_process(func):
     def wrapper(stopped, error, *args):
         try:
             func(stopped, error, *args)
-        except Exception as e:
-            traceback.print_exc()
```

Comment:
this is no longer needed ?

Reply:
No. `logger.info` will print exception when `exc_info` is passed.